### ISSUE TITLE
Add A Dev Tools Module

### ISSUE THREAD

**Issue by ryanjbaxter** [spring-cloud/spring-cloud-kubernetes#394]
The module would help with the developer experience in building and debugging Kubernetes based apps.

To start with I would like to add some support for using Telepresence, it could
* Automatically add env vars from JSON file generated when running Telepresnece
* Automatically configure the Fabric8IO client to use certs for connecting to the remote cluster

**Comment by geoand** [comment]
@salaboy you have used Telepresence, correct?

**Comment by salaboy** [comment]
@geoand yes I have, not with spring cloud k8s in the class path.. the main problem usually is the difference from running from inside the cluster and from outside due RBAC policies applied internally to access Kubernetes services. 
As I mentioned yesterday, I think that we need to try and compare Squash as well:
https://squash.solo.io

**Comment by wind57** [comment]
this is another issue that IMO we can close. `Telepresence` is well and fine and can do all of the things that someone would need via a dev tools, like:

- connect local machine to the remote cluster
- intercept a workload
- export remote env vars with --env-file or --env-json
- run locally or even via --docker-run / --docker-build
- handle the cluster-side networking model on its own

@ryanjbaxter wdyt?
